packages/Badge/package.json

```diff
@@ -7,7 +7,7 @@
   "description": "",
   "main": "./lib/index.js",
   "scripts": {
-    "build": "../../node_modules/.bin/tsc --jsx react --esModuleInterop ./src/* --outDir lib ",
+    "build": "../../node_modules/.bin/rimraf ./lib && ../../node_modules/.bin/tsc --jsx react --esModuleInterop ./src/*.tsx --outDir lib ",
```

Comment:
We should add this to all the packages in the library or find a better way to handle it.